builder/chroot-script.sh

```diff
@@ -1,51 +1,172 @@
 #!/bin/bash
 set -ex
 
+# save HYPRIOT_* content from /etc/os-release. Needs to be done in
+# case base-files gets updated during build
+< /etc/os-release grep HYPRIOT_ > /tmp/os-release.add
```

Comment:
I guess this workaround isn't needed anymore with os-rootfs 1.1.1, so I think you should revert it.